src/common/pp-ga/events.ts

```diff
@@ -152,9 +153,20 @@ export function annotationFormMoved() {
   });
 }
 
+// Annotation Request events
+
+export function annotationRequestLinkClicked(url: string) {
+  sendEventByMessage({
+    eventCategory: 'AnnotationRequest', eventAction: 'ClickRe', eventLabel: 'AnnotationRequestLinkClicked',
+    [GACustomFieldsIndex.eventUrl]: url,
+  });
+}
+
+// Annotation Upvote events
+
 export function annotationUpvoted(annotationId: string, priority: string, isCommentBlank: boolean, link: string) {
   sendEventByMessage({
-    eventCategory: 'Annotation', eventAction: 'Upvote', eventLabel: 'AnnotationUpvoted',
+    eventCategory: 'AnnotationUpvote', eventAction: 'Add', eventLabel: 'AnnotationUpvoted',
```

Comment:
Nice! I just noticed when playing with GA panel, that this name was too vague and should probably be no wider than an action/feature.